src/components/SortBySelector.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import { useTheme } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Drawer from '@material-ui/core/Drawer';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import CakeOutlinedIcon from '@material-ui/icons/CakeOutlined';
+import WbSunnyOutlinedIcon from '@material-ui/icons/WbSunnyOutlined';
+import StarOutlinedIcon from '@material-ui/icons/StarOutlined';
+
+export const SortEnum = {
+  ACCOUNT_AGE: 'account age',
+  ACCOUNT_KARMA: 'account karma',
+  AWARD_COUNT: 'award count',
+};
+
+const useStyles = makeStyles((theme) => {
+  return {
+    root: {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    formControl: {
+      '&. MuiInputBase--root': {},
+      '&:hover': {
+        backgroundColor: 'transparent',
+      },
+      '& .Mui-focused': {
+        backgroundColor: 'transparent',
+      },
+    },
+    button: {
+      '&:hover': {
+        backgroundColor: 'transparent',
+      },
+    },
```

Comment:
We can customize the theme in `src/styles/index.js`

```
const config = {
  direction: 'ltr',
  typography,
  overrides: {
  MuiAlert: {
    icon: {
      alignSelf: 'center',
    },
  },
  MuiAppBar: {
    colorPrimary: {
      backgroundColor: pipe17.palette.primary.main,
    },
  },
  MuiButton: {
    root: {
      textTransform: undefined,
    },
    containedSecondary: {
      backgroundColor: pipe17.palette.secondary.main,
    },
  },
  },

```

In dev mode you can figure out the class names by inspecting in dev tools, eg `MuiButton-root`

Reply:
https://material-ui.com/customization/components/#global-theme-override

Reply:
@LadyMartel FYI ^ feel free to follow up but im excited to get this merged

Reply:
TY! I will take a look :D